doc/whats_new.rst

```diff
@@ -40,6 +40,8 @@ Changelog
 
     - Add reading of Elekta ``xfit`` fixed-position dipole time courses using :func:`mne.read_dipole` by `Eric Larson`_.
 
+    - Add functionality to :class:`mne.decoding.GeneralizationAcrossTime` to load sklearn `scorers` by passing string in parameter by `Asish Panda`_.
```

Comment:
" - Accept :class:`mne.decoding.GeneralizationAcrossTime`'s ``scorer`` parameter to be a string that refers to a scikit-learn metric scorer, by `Asish Panda`_."

(use double quotes for scorer, as in ``xfit `` on the above line)